src/chrome/content/rules/MNGinteractive.com.xml

```diff
@@ -9,12 +9,12 @@
 		- extras	(redirects to http.../$, akamai)
 
 -->
-<ruleset name="MNGinteractive.com">
+<ruleset name="MNGinteractive.com" default_off="breaks layout">
```

Comment:
The issue is, that the host times out. So could you use `default_off="timeout"`?